tests/serialization/test_dag_serialization.py

```diff
@@ -978,6 +978,107 @@ def check_task_group(node):
 
         check_task_group(serialized_dag.task_group)
 
+    def test_deps_sorted(self):
+        """
+        Tests serialize_operator, make sure the deps is in order
+        """
+        from airflow.operators.dummy import DummyOperator
+        from airflow.sensors.external_task import ExternalTaskSensor
+
+        execution_date = datetime(2020, 1, 1)
+        with DAG(dag_id="test_deps_sorted", start_date=execution_date) as dag:
+            task1 = ExternalTaskSensor(
+                task_id="task1",
+                external_dag_id="external_dag_id",
+                mode="reschedule",
+            )
+            task2 = DummyOperator(task_id="task2")
+            task1 >> task2
+
+        serialize_op = SerializedBaseOperator.serialize_operator(dag.task_dict["task1"])
+        deps = serialize_op["deps"]
+        assert len(deps) == 5
+        assert deps[0] == 'airflow.ti_deps.deps.not_in_retry_period_dep.NotInRetryPeriodDep'
+        assert deps[1] == 'airflow.ti_deps.deps.not_previously_skipped_dep.NotPreviouslySkippedDep'
+        assert deps[2] == 'airflow.ti_deps.deps.prev_dagrun_dep.PrevDagrunDep'
+        assert deps[3] == 'airflow.ti_deps.deps.ready_to_reschedule.ReadyToRescheduleDep'
+        assert deps[4] == 'airflow.ti_deps.deps.trigger_rule_dep.TriggerRuleDep'
```

Comment:
```suggestion
        assert deps == [
            'airflow.ti_deps.deps.not_in_retry_period_dep.NotInRetryPeriodDep',
            'airflow.ti_deps.deps.not_previously_skipped_dep.NotPreviouslySkippedDep',
            'airflow.ti_deps.deps.prev_dagrun_dep.PrevDagrunDep',
            'airflow.ti_deps.deps.ready_to_reschedule.ReadyToRescheduleDep',
            'airflow.ti_deps.deps.trigger_rule_dep.TriggerRuleDep',
        ]
```

Same for similar assert statements below.

Reply:
fixed